docs/AWS_DEV_SETUP_GUIDE_KR.md

```
@@ -0,0 +1,1850 @@
+> 저비용 + 운영 환경과 동일한 구조로 개발 서버 구축하기
+
+## 목차
+
+1. [사전 준비](#1-사전-준비)
+2. [VPC 및 보안 그룹 설정](#2-vpc-및-보안-그룹-설정)
+3. [RDS MySQL 생성](#3-rds-mysql-생성)
+4. [ElastiCache Redis 생성](#4-elasticache-redis-생성)
+5. [ECR 리포지토리 생성](#5-ecr-리포지토리-생성)
+6. [Route 53 도메인 설정](#6-route-53-도메인-설정)
+7. [EC2 인스턴스 생성 및 설정](#7-ec2-인스턴스-생성-및-설정)
+8. [애플리케이션 배포](#8-애플리케이션-배포)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)